benchmarks/bench_numba_compile.py

```diff
@@ -0,0 +1,37 @@
+import importlib
+
+import interegular
+import numba
+
+import outlines
+
+from .common import clear_outlines_cache, setup_tokenizer
+
+outlines.disable_cache()
```

Comment:
We need to contain global state changes like this in the set-up/tear-down of these benchmarks, or&mdash;better yet&mdash;avoid them completely.